### ISSUE TITLE
One Click Login Issues

### ISSUE THREAD

**Issue by XtrmePlaya** [atomizer/muledump#90]
I am having issues with this and I can not figure out why.
I installed Autoit,
I changed mulelogin in accounts.js to 1,
When i try to run lib/mulelogin.au3, This pops up every time, and i am a system administrator.
http://puu.sh/mCBJi/ff74a306cb.png

I would Love some help please!

~Tyler

**Comment by atomizer** [comment]
This is strange. Try this: right click -> compile script, then right click the created .exe -> run as administrator.

**Comment by XtrmePlaya** [comment]
That seemed to work, But when I click the button to open from muledump. It opens the website, but it is not logged in. The usernames and passwords are all correct.

**Comment by Nightfirecat** [comment]
Based on this issue and #91, maybe the flash storage location for realm changed? Haven't actually booted the game and tested recently or anything, but just a guess.

**Comment by XtrmePlaya** [comment]
How can i fix this then?

**Comment by Nightfirecat** [comment]
Could you try clearing your Macromedia Flash local cache, then load up RotMG and find out what directory is re-created?

**Comment by XtrmePlaya** [comment]
I can probably do that. Once i figure out how to. :P

**Comment by XtrmePlaya** [comment]
the directory created in "local storage settings by site" is realmofthemadgodhrd.appspot.com
Which is part of the link i use to open Realm in Flash Player. 

The link i use to open Realm in flash is,
https://realmofthemadgodhrd.appspot.com/AssembleeGameClient1452042685.swf

**Comment by Nightfirecat** [comment]
Well, by the look of things, [that seems to be getting populated already](https://github.com/atomizer/muledump/blob/master/lib/mulelogin.au3#L111). Can you try confirming if the file changes after using one-click on different accounts? (it should insert your account username/password among other details into the file)

**Comment by XtrmePlaya** [comment]
It does not seem to change

**Comment by XtrmePlaya** [comment]
Any update?

**Comment by Pyrolizrd** [comment]
Uh, I had this issue, I backed up my accounts.js file, uninstalled muledump, and then reinstalled it. I then hand opened each mule-account on flash projector, thus creating the necessary rotmg.sol file, enabled one-click login, and voila: it worked! My problem was I generally play in chrome, but apparently muledump grabs the .sol file from the the Macromedia flash player not pepperflash, so you had to open each account on the flash projector.

**Comment by atomizer** [comment]
Is this still an issue for you @XtrmePlaya?

**Comment by XtrmePlaya** [comment]
Did not seem to help, but i stopped playing for a while. It just keeps
opening blank rotmg games.

**Comment by quzzimi** [comment]
Edit mulelogin.au3 like this  if u got an "unable to write registry" 
complile and run with admin will not load the player data , only open the flashlayer.exe
#RequireAdmin
#include <String.au3>
#include <File.au3>
#include <Array.au3>    
this fix the problem for me

**Comment by Nightfirecat** [comment]
I'm not sure I understand&mdash;what did you add, the `#RequireAdmin` line at the top of the file?

**Comment by quzzimi** [comment]
yes , it fix the shitty " admin thingly" and execute the au3 file directly not the complied exe

**Comment by quzzimi** [comment]
unable to write registry happend in win 8 and win 10 
win xp doesnt cause it weak reg protection

**Comment by TheIvyX** [comment]
When I use the one-click login feature on Chrome and login to an account there, if I reopen Flash Player the account that I logged in onto Chrome will login to Flash Player as well. If log into 2 accounts manually  into Chrome and into Flash Player, they will be separate.

Here's an example, I log into Account 1 onto Chrome and it will keep me logged in Account 1 the next time I open Chrome up. I log into Account 2 onto Flash Player and it will keep me logged in Account 2 the next time I open Flash Player up, but if I open Chrome, I will still be logged on Account 1. With Muledump, if I use the one-click login to log into Account 1, it will open up a new window in Chrome. If I open up Flash Player, it will also log me into Account 1. Is there a way to prevent the one-click login feature to log into Flash Player?

**Comment by quzzimi** [comment]
Chrome use it own internal Flash player , nothing to do with your own system flash player
Muledume use the window flash player 
 Is there a way to prevent the one-click login feature to log into Flash Player? => defaul web browser = Chrome

**Comment by atomizer** [comment]
disable chrome's built-in player (and install adobe flash plugin for chrome if not installed) https://www.enounce.com/chrome-disable-pepperflash
